src/components/CommunityPartners/CommunityPartners.vue

```diff
@@ -0,0 +1,260 @@
+          extraClass: "",
+        },
+        {
+          name: "intlify",
+          image: "https://github.com/intlify.png",
+          link: "https://intlify.dev/",
+          description:
+            "Intlify is a project that aims to improve Developer Experience in software internationalization. We will aim to improve the Developer Experience of internationalization by providing libraries, frameworks, and tools that break down barriers to internationalization.",
+          extraClass: "rounded-full",
+        },
+        {
+          name: "Code for Japan",
+          image: "/assets/code_for_japan.jpg",
+          link: "https://www.code4japan.org/",
+          description:
+            "Code for Japan is one of the largest communities of civic hackers in Japan. We organize hackathons to improve and promote civic tech and collaborate with local governments and communities.",
+          extraClass: "rounded-full",
+        },
+      ],
+      isModalOpen: false,
+      isModalLoading: false,
+      isModalSuccessful: false,
+      isModalFailed: false,
+      sheetDbBaseUrl: 'https://sheetdb.io/api/v1/',
+      sheetDbConfig: {
+        address: import.meta.env.PUBLIC_SHEETDB_ADDRESS,
+        auth_login: import.meta.env.PUBLIC_SHEETDB_LOGIN,
+        auth_password: import.meta.env.PUBLIC_SHEETDB_PASSWORD,
+      },
+    };
+  },
+  methods: {
+    toggleModal: function () {
+      const htmlEl = document.documentElement;
+      const headerEl = document.querySelector("#app");
+
+      if (this.isModalOpen) {
+        htmlEl.classList.remove("overflow-y-hidden");
+        headerEl.classList.remove("hidden");
+      } else {
+        htmlEl.classList.add("overflow-y-hidden");
+        headerEl.classList.add("hidden");
+      }
+
+      this.isModalOpen = !this.isModalOpen;
+    },
+    fetchSendPostData: async function (url = '', data = {}) {
+      const response = await fetch(url, {
+        method: 'POST',
+        headers: new Headers({
+            'Content-Type': 'application/json',
+            'Authorization': `Basic ${btoa(`${this.sheetDbConfig.auth_login}:${this.sheetDbConfig.auth_password}`)}`
+        }),
+        body: JSON.stringify(data)
+      });
+      if (response.ok) return response.json();
+      throw new Error(response.status);
+    },
+    sheetDBCreateRow: async function(address, data) {
+      return this.fetchSendPostData(this.sheetDbBaseUrl + address, {
+        data: { ...data }
+      });
+    },
+    handleFormSubmit: async function ({
+        founderName,
+        communityName,
+        website,
+        email,
+        whyPartnerWithUs,
+        howYouHearAboutUs
+    }) {
+      this.isModalLoading = true;
+
+      this.sheetDBCreateRow(this.sheetDbConfig.address, {
+          'Founder Name': founderName,
```

Comment:
Don't you think we should use `Your name` here?
Imagine I submit this form, but I'm not the founder. 😃

```suggestion
          'Your Name': founderName,
```

Not sure if I should change the variable; you might need to change that in other parts of the code. 😅

Reply:
I think we should ask @vinzvinci I just followed what he wrote in #194.

Reply:
> I think we should ask @vinzvinci, I just followed what he wrote in #194.

Founder Name is best.